lib/balance-tx/test/spec/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -602,39 +602,39 @@ spec_balanceTx era = describe "balanceTx" $ do
         -- have separate 'it' statements for different expectations of the same
         -- test case.
         let nPayments = 10
-        let paymentOuts = replicate nPayments $
-                W.TxOut
-                    dummyAddr
-                    (W.TokenBundle.fromCoin (W.Coin 1_000_000))
+        let paymentOuts :: [TxOut era]
+            paymentOuts = replicate nPayments $
+                mkBasicTxOut dummyAddr (ada 1)
+
         let ptx = paymentPartialTx paymentOuts
 
         -- True for values of nPayments small enough not to cause
         -- 'ErrBalanceTxMaxSizeLimitExceeded' or ErrMakeChange
         let nChange = max nPayments 1
         let changeState0 = DummyChangeState 0
-        let expectedChange = fmap Convert.toWalletAddress <$>
+        let expectedChange =
                 flip evalState changeState0
                 $ replicateM nChange
                 $ state @Identity dummyChangeAddrGen.genChangeAddress
 
         let
-            address :: IsRecentEra era => TxOut era -> W.Address
-            address = Convert.toWallet . view addrTxOutL
+            address :: TxOut era -> Address
+            address = view addrTxOutL
```

Comment:
this is reuse from the ledger's https://github.com/IntersectMBO/cardano-ledger/blob/75735807c05df234fb6f66cd681eacf534937f1b/libs/cardano-ledger-core/src/Cardano/Ledger/Core.hs#L323 ?